### PR TITLE
corrected path for mentor-guide.md

### PR DESCRIPTION
**Pull request by dineshkumar181094**
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by dineshkumar181094** [comment]
Sorry for the multiple PR, raising the first PR

File: mentoring/programs/the1-on-1hour.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ We will attempt to match you with the closest mentor of your area of interested
 
 ## Mentors
 
-If you are interested in signing up to be a mentor for this, please see the [mentor guidelines](mentor-guide.md).
+If you are interested in signing up to be a mentor for this, please see the [mentor guidelines](../processes/mentor-guide.md).
```
Comment (mrbobbytables) [inline mentoring/programs/the1-on-1hour.md:26]:
```suggestion
If you are interested in signing up to be a mentor for this, please see the [mentor guidelines](/mentoring/processes/mentor-guide.md).
```
Using the full path from the root is preferred. It ensures if the doc is ever moved again the link won't have to be fixed.

Comment (dineshkumar181094) [reply mentoring/programs/the1-on-1hour.md]: yes, make sense.
Update the PR.

**Comment by dineshk-hotstar** [comment]
Are we merging this?

**Comment by dineshkumar181094** [comment]
@nikhita can you merge this PR.

**Comment by mrbobbytables** [comment]
@dineshkumar181094 were you going to roll in your other PRs into this one?

**Comment by dineshkumar181094** [comment]
> @dineshkumar181094 were you going to roll in your other PRs into this one?

Nope

**Pull request merged**